bpfman/src/lib.rs

```diff
@@ -9,14 +9,23 @@ use std::{
 
 use aya::{
     programs::{
-        fentry::FEntryLink, fexit::FExitLink, kprobe::KProbeLink, links::FdLink, loaded_programs,
-        trace_point::TracePointLink, uprobe::UProbeLink, FEntry, FExit, KProbe, TracePoint, UProbe,
+        fentry::FEntryLink,
+        fexit::FExitLink,
+        kprobe::KProbeLink,
+        links::FdLink,
+        loaded_programs,
+        tc::{SchedClassifierLink, SchedClassifierLinkId, TcAttachOptions},
+        trace_point::TracePointLink,
```

Comment:
where is `tcx` in this ?

Reply:
A tcx and tc programs are considered to be the same type by Aya and the kernel.  It's the attach type that's different.